packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -0,0 +1,654 @@
+---
+title: Authenticators Admin API
+category: management
+---
+
+# Authenticators Admin API
+
+Authenticator Admin API enables an Org Administrator to configure which authenticators are available to end users to be used for signing into applications.
+
+End users would be required to use one or more authenticators depending on the security requirements of the application sign-on policy. 
+
+IDX currently supports authenticators for the following:
+
+**Knowledge based:**
+* Password
+* Security Question
+
+**Possession based:**
+* Phone (supports SMS, Call)
+* Email
+* WebAuthn 
```

Comment:
Thinking about this more, should we call this `Security Key` and say `(supports WebAuthn)`?